### DISCUSSION TITLE
delay not working properly

### DISCUSSION THREAD

**Discussion by jacopisky** [SpenceKonde/ATTinyCore#683] [Support / Q & A]
the delay macro of the arduino IDE on ATtiny2313A-SU is working but wrong time delay(1000) gives a real delay of about 20 seconds

**Comment by SpenceKonde** [comment]
You have selected a clock speed that doesn't match the clock speed thatcthe
chip is running at, and/or have not done burn bootloader to write the fuses
to select the desired clock source. See the faq, this is universal behavior
on classic avrs.

____________
Spence Konde
Azzy’S Electronics

New products! Check them out at tindie.com/stores/DrAzzy
GitHub: github.com/SpenceKonde
ATTinyCore: Arduino support for almost every ATTiny microcontroller
Contact: ***@***.***
